app/models/profile.rb

```diff
@@ -33,6 +33,7 @@ def decrypt_pii(user_access_key)
   end
 
   def recover_pii(personal_key)
+    personal_key = personal_key.tr('-', ' ')
```

Comment:
Since this method is called after a form submission, I would propose to move `PersonalKeyForm#valid_personal_key?` to a `PersonalKeyValidator` in `app/validators`, and include it in `PersonalKeyForm` and in `ReactivateProfileForm`. That way, we know the input has been validated to be the right format before it gets to the `recover_pii` method.